larva/src/main/java/nl/nn/adapterframework/testtool/TestTool.java

```diff
@@ -3072,12 +3072,16 @@ private static String getEncoding(String fileName, String content) {
 	}
 
 	public static int compareResult(String step, String stepDisplayName, String fileName, String expectedResult, String actualResult, Properties properties, Map<String, Object> writers, String queueName) {
+		if(properties.getProperty(queueName + ".ignoreRead") != null && properties.getProperty(queueName + ".ignoreRead").equalsIgnoreCase("true") ){
```

Comment:
As I understand, the idea of this solution is that configuration will be something like:
```
step1.manage.db.write = scenario01/md-delete.xml
step2.manage.db.read = scenario01/md-delete-result.xml
step2.manage.db.ignoreRead = true
step3.java....
```
Although this provides the requested functionality, it is a bit clumsy: it requires a 'expected file' that is never actually compared to the 'actual file'. I think this is confusing.

I would like to propose to 
- rename the key 'ignoreRead' into 'differences', that can have the folloing values:
  - fail: the default, as it is now: show the differences, and fail the test if they exist
  - ignore: show the differences, allow to update the expected, but do not fail the test
  - ignore-silent: do not even show the differences
- add a key 'readIgnore' that can be set to true, to will act as a combination of 'read' and 'differences=ignore-silent'